e2e/config/android.conf.js

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+const {join} = require('path');
+const {config} = require('./shared.conf');
+
+/* eslint-disable no-process-env */
+
+require('dotenv').config();
+const {
+    DEV,
+    FULL_RESET,
+    NO_RESET,
+    ANDROID_APP_FILE_PATH,
+    ANDROID_APP_PACKAGE,
+    ANDROID_DEVICE_NAME,
+    ANDROID_PLATFORM_VERSION,
+    REMOTE_TEST_SERVER_URL,
+} = process.env;
+
+// ============
+// Specs
+// ============
+config.specs = [
+    './test_android/**/*_spec.js',
+];
+
+// ============
+// Suites
+// For grouping test specs
+// ============
+config.suites = {
+    login: ['./test_android/login/**/*_spec.js'],
+    messaging: ['./test_android/messaging/**/*_spec.js'],
+};
+
+// ============
+// Capabilities
+// ============
+// For all capabilities, refer to:
+// http://appium.io/docs/en/writing-running-appium/caps/
+config.capabilities = [
+    {
+
+        // Default values for Android
+        platformName: 'Android',
+        maxInstances: 1,
+
+        // For W3C, the Appium capabilities need to have an extension prefix of `appium:`
+        // http://appium.io/docs/en/writing-running-appium/caps/
+
+        // Should match with connected Android device or emulator
+        'appium:deviceName': ANDROID_DEVICE_NAME,
+        'appium:platformVersion': ANDROID_PLATFORM_VERSION,
```

Comment:
I get this are env vars and they can be easily changed, but can we do this in the same way the rn-cli does it? Basically it should pick the first available Sim/Emu if none is defined

Reply:
I agree and it's also our end goal.  Will try to see on how it could be done and will try to include in this PR but can't find an easy workaround, I hope it's non-blocking and can be addressed in a separate PR.